react-design/react-design-app/src/RotatingBanner.jsx

```diff
@@ -0,0 +1,41 @@
+export default function RotatingBanner({ items }) {
+  function Banner() {
```

Comment:
In the example we went over on Monday, the solution was all in one file. In this case, we should have each React Component in its own function instead of being the children of Rotating Banner. 

However, it is even better practice if each component is its own file and we import/export as needed as we did in prior exercises.

Reply:
Please update with one of the two options above!